lab_3_keywords_textrank/main.py

```diff
@@ -278,7 +341,12 @@ def calculate_inout_score(self, vertex: int) -> int:
                 number of incidental vertices
         If vertex is not present in the graph, -1 is returned
         """
-        pass
+        if vertex in self._matrix_vertex:
+            for elem in self._matrix_vertex:
+                if elem == vertex:
+                    return sum(string for string in self._matrix[self._matrix_vertex.index(elem)])
```

Comment:
do you really need this for? or just `sum(self._matrix[self._matrix_vertex.index(elem)])`